services/console/src/pages/[lang]/docs/reference/architecture.astro

```diff
@@ -67,4 +68,5 @@ const modified = "2024-02-06T07:56:00Z";
 <DocsLayout title={architecture(lang)} description={description()} heading={heading()} published={published} modified={modified} lang={lang} collection={collection}
 slug="architecture" canonical="https://bencher.dev/docs/reference/architecture/">
   <ArchitecturePanel />
+  <ERDPanel />
```

Comment:
Go ahead and create another file for this: `services/console/src/pages/[lang]/docs/reference/schema.astro`